react/features/conference/components/Conference.native.js

```diff
@@ -264,12 +286,38 @@ function _mapDispatchToProps(dispatch) {
  * @param {Object} state - The Redux state.
  * @private
  * @returns {{
+ *     _connecting: boolean,
  *     _toolboxVisible: boolean
  * }}
  */
 function _mapStateToProps(state) {
+    const { connecting, connection } = state['features/base/connection'];
+    const { joining, conference, leaving } = state['features/base/conference'];
+
+    // The logic here looks a bit twisted. We try to avoid flapping of our flag,
```

Comment:
@lyubomir do you have any suggestions to make this nicer? I couldn't come up with a better way to express it.